WSL/faq.md

```diff
@@ -28,53 +31,76 @@ You can also access your local machine’s filesystem from within the Linux Bash
 
 ## Why would I use, for example, Ruby on Linux instead of on Windows?
+
 Some cross-platform tools were built assuming that the environment in which they run behaves like Linux. For example, some tools assume that they are able to access very long file paths or that specific files/folders exist. This often causes problems on Windows which often behaves differently from Linux.
 
 Many languages like Ruby and node are often ported to, and run great, on Windows. However, not all of the Ruby Gem or node/NPM library owners port their libraries to support Windows, and many have Linux-specific dependencies. This can often result in systems built using such tools and libraries suffering from build and sometimes runtime errors or unwanted behaviors on Windows.
 
 These are just some of issues that caused many people to ask Microsoft to improve Windows’ command-line tools and what drove us to partner with Canonical to enable native Bash and Linux command-line tools to run on Windows.
 
 ## What does this mean for PowerShell?
+
 While working with OSS projects, there are numerous scenarios where it’s immensely useful to drop into Bash from a PowerShell prompt. Bash support is complementary and strengthens the value of the command-line on Windows, allowing PowerShell and the PowerShell community to leverage other popular technologies.
 
 Read more on the PowerShell team blog -- [Bash for Windows: Why it’s awesome and what it means for PowerShell](https://blogs.msdn.microsoft.com/powershell/2016/04/01/bash-for-windows-why-its-awesome-and-what-it-means-for-powershell/)
 
 ## Can I run ALL Linux apps in WSL?
-No! WSL is a tool aimed at enabling users who need them to run Bash and core Linux command-line tools on Windows. 
+
+No! WSL is a tool aimed at enabling users who need them to run Bash and core Linux command-line tools on Windows.
 
 WSL does **not** aim to support GUI desktops or applications (e.g. Gnome, KDE, etc.)  
 
 Also, even though you will be able to run many popular server applications (e.g. Redis), we do not recommend WSL for hosting production services – Microsoft offers a variety of solutions for running production Linux workloads in Azure, Hyper-V, and Docker. 
 
 ## What Windows SKUs is WSL included in?
+
 Windows Subsystem for Linux is available on the desktop version of Windows for Windows 10 Anniversary and Creators update or later.
 
 Beginning in the Fall Creators update WSL will be available on both the desktop and server SKUs of Windows.
 
 ## What processors does WSL support?
+
 WSL supports x64 and ARM CPUs.
 
 ## How do I access my C: drive?
-Mount points for hard drives on the local machine are automatically created and provide easy access to the Windows filesystem. 
- 
+
+Mount points for hard drives on the local machine are automatically created and provide easy access to the Windows filesystem.
+
 **/mnt/\<drive letter>/**
- 
+
 Example usage would be `cd /mnt/c` to access c:\
 
+## How do I set up Git Credential Manager?

```

Comment:
Possible alternative title: 'How do I make WSL use my Windows git permissions?'

That title is often searched / asked with that wording on Twitter etc.